content/en/continuous_integration/tests/ruby.md

```diff
@@ -368,15 +372,67 @@ Datadog.configure do |c|
   # The name of the service or library under test
   c.service = 'my-ruby-app'
 
-  # Enables the RSpec instrumentation
-  c.ci.instrument :rspec
+  c.ci.instrument :minitest
 end
 ```
 
-Run your tests as you normally do, specifying the environment where test are being run (for example, `local` when running tests on a developer workstation, or `ci` when running them on a CI provider) in the `DD_ENV` environment variable. For example:
+Run your tests as you normally do, specifying the environment where tests are being run in the `DD_ENV` environment variable.
+
+You could use the following environments:
+
+* `local` when running tests on a developer workstation
+* `ci` when running them on a CI provider
+
+For example:
 
 ```bash
-DD_ENV=ci bundle exec rake spec
+DD_ENV=ci bundle exec rake test
+```
+
+{{% /tab %}}
+
+{{% tab "Cucumber" %}}
+
+The Cucumber integration traces executions of scenarios and steps when using the `cucumber` framework.
+
+To activate your integration, add the following code to your application:
```

Comment:
Just to double-check (and for my education), for Cucumber support, the `test_helper.rb` file is not relevant?

Reply:
Cucumber could be configured in different ways depending on which other test libraries are used, I think this is why the exact configuration file is omitted here. This doc will change anyway a lot before GA